### PR TITLE
PDFToolBox

### PR DESCRIPTION
**Pull request by shrehs**
Closes: #215 

- **Title**:  Completion of PDFToolbox 
- **Your Name**: Shreya H S
- **Idenitfy yourself**:  GSSOC-Ext, Hacktoberfest

### Describe the add-ons or changes you've made 📃
Implement mergePDF, splitPDF, rotatePDF, encryptPDF, decryptPDF functions

Give a clear description of what have you added or modifications made

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/Contributing.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/Jarvis/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### Screenshots 📷

[PDFToolbox1.webm](https://github.com/user-attachments/assets/b22cb462-f538-4f1b-823a-f384d41dc049)
Screenshot (144)
Screenshot (145)

### CODE REVIEW

**Comment by Zedoman** [comment]
Also provide a video for that @shrehs

**Comment by shrehs** [comment]
> Also provide a video for this @shrehs

I have! It is just above the image (underlined blue font)

**Review by Zedoman — DISMISSED** [review]

File: src/apps/pages/programs/ApiPrograms/recipe_finder/.gitignore
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/ApiPrograms/recipe_finder/.gitignore:1]: .

File: src/apps/pages/programs/ApiPrograms/recipe_finder/recipe_finder.py
Comment (Avdhesh-Varshney) [inline src/apps/pages/programs/ApiPrograms/recipe_finder/recipe_finder.py:1]: .

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Remove these 2 files from your commit

**Comment by shrehs** [comment]
> Remove these 2 files from your commit

Done

**Review by Avdhesh-Varshney — APPROVED** [review]

**Review by Zedoman — APPROVED** [review]

**Pull request merged**